packages/amplify_core/lib/src/hub/amplify_hub_impl.dart

```diff
@@ -85,6 +85,10 @@ class AmplifyHubImpl extends AmplifyHub {
     Future.wait<void>([
       for (final stream in _availableStreams.values) stream.close(),
     ]).ignore();
+    Future.wait<void>([
```

Comment:
Missed this from before, but what does this block do?

Reply:
No worries, this was just added. @dnys1 helped debug this one. It cleans up hub event subscriptions on cancel, an issue I was seeing in testing.

Reply:
cool, thanks for info. Might be worth a small comment here to explain.